### DISCUSSION TITLE
micro UI to old UI redirect and vice versa

### DISCUSSION THREAD

**Discussion by rishabhsuriwtc** [egovernments/Digit-Core#54] [DIGIT Platform]
is there any shortcut/link/button to redirect to the micro ui from the old ui ? i saw the option to come back to the micro ui from old ui on the home screen of old ui which lands us back to the listing screen of the micro ui. but if i just want to use the reports from the old ui, is there any work around to easily come back to the home page of micro ui ?

**Comment by jagankumar-egov** [comment]
For navigation between old and new UI check this documentation, 
This feature is available from Release 2.8
https://urban.digit.org/platform/configure-digit/configuring-digit-services/common-ui-docs/navigation-between-old-and-new-ui

`The simple way to get the new UI link in the old UI is to add an action to navigate to the new UI and assign the required role to whom it has to show that link.`
